man/machinery-man.1.md

```diff
@@ -0,0 +1,10 @@
+
+## Man — Shows man page
```

Comment:
I wasn't precise enough what I meant when I said it's a head line:  `## man — Shows Man Page`.